cpp/include/bindings/bindings_renderer.hpp

```diff
@@ -100,6 +100,22 @@ extern "C"
 	JNIEXPORT void JNICALL
 	Java_app_rive_runtime_kotlin_renderers_RendererSkia_cppClearSurface(
 	    JNIEnv*, jobject, jlong rendererRef);
+
+	JNIEXPORT void JNICALL
+	Java_app_rive_runtime_kotlin_renderers_RendererSkia_cppSave(
+	    JNIEnv*, jobject, jlong rendererRef);
+
+	JNIEXPORT void JNICALL
+	Java_app_rive_runtime_kotlin_renderers_RendererSkia_cppRestore(
+	    JNIEnv*, jobject, jlong rendererRef);
+
+	JNIEXPORT jint JNICALL
+	Java_app_rive_runtime_kotlin_renderers_RendererSkia_cppWidth(
```

Comment:
Mmh, since we're rendering to a surface, the fact that it "knows" how big it is should be ok.
We can eventually clean it up and get it all from the `EGLThreadState`, if needed.